v1/plugin/stream_collector_proxy.go

```diff
@@ -2,86 +2,130 @@ package plugin
 
 import (
 	"fmt"
+	"time"
 
 	"golang.org/x/net/context"
 
 	"github.com/intelsdi-x/snap-plugin-lib-go/v1/plugin/rpc"
 )
 
+const defaultMaxCollectDuration = 10 * time.Second
```

Comment:
It is default zero but there is no need to declare that in this way (variable of type int64 equals 0 by default)